edge/pkg/apis/dmi/v1/api.proto

```diff
@@ -0,0 +1,298 @@
+syntax = "proto3";
+
+//option go_package = "path;name";
+option go_package="./;v1";
+package v1;
+
+import "google/protobuf/any.proto";
+
+service DeviceManagerService {
```

Comment:
I think we should add some comments for the service and interface

Reply:
i have added the comments for interfaces.